model/lti.go

```diff
@@ -48,9 +51,9 @@ type LMS interface {
 }
 
 type LTISettings struct {
-	Enable                    bool
-	EnableSignatureValidation bool
-	LMSs                      []interface{}
+	Enable                    bool          `json:"enable"`
+	EnableSignatureValidation bool          `json:"enablesignaturevalidation"`
+	LMSs                      []interface{} `json:"lmss"`
```

Comment:
So as per my previous comment, I don't think adding the json format string to this struct is necessary and that will leave the json properties the same as the struct field names which is preferable.

Reply:
For the reason mentioned above, since the plugin property can only be lowercase, we need this change.